client-app/shared/checkout/components/checkout-section.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    class="bg-white lg:mb-6 lg:rounded"
+    :class="[isProductsBlock ? 'shadow lg:pb-11' : 'shadow-inner pb-8 lg:shadow']"
```

Comment:
Can we apply classes to the component where we use it instead of isProductBlock usage?

Reply:
@trueboroda The property is used in two places so we can't use $attrs.class